.github/renovate.json5

```
@@ -114,7 +114,7 @@
     },
     {
       customType: 'regex',
-      fileMatch: ['*.md$'],
+      fileMatch: ['.*.md$'],
```

Does that mean we need

```suggestion
      fileMatch: ['.*\.md$'],
```

---

Prettier nukes them

---

```suggestion
      fileMatch: ['.*\\.md$'],
```

---

probably